src/content/guides/pay-payment-app-integration.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Paying a Payment Request
+description: Integrate your app with Centrapay to pay payment requests on merchant terminals and POS
+nav:
+  path: Reference/App Integrations
+  order: 1
+---
+
+## Overview
+
+This guide is for a [Third-party Asset Provider](/guides/integrating-third-party-asset) to enable paying with their
+digital assets via Centrapay.
+
+## Authenticating API calls on behalf of users
+
+Third-party applications can make API calls on behalf of users with Centrapay
+API keys. All API requests to Centrapay must be made from the integrator’s backend.
+Centrapay API keys must also be managed on the backend and stored as secrets to
+minimize the risk of API keys being compromised.
+
+Seperate API keys will be used for test and live payments.
+
+> Contact [integrations@centrapay.com](mailto:integrations@centrapay.com) to be issued your API keys.
+
+## Payment Flow Implementation
+
+The payment flow below describes how a Third-party Asset Provider should integrate
+with the [Dynamic Merchant QR Code flow](/guides/payment-flows#dynamic-merchant-qr-code) to enable digital payments within their
+application. Below is an overview of the process, followed by a more in-depth
+look at the integrator’s responsibilities.
+
+> Note this is assuming the best case scenario, and not accounting for edge cases.<br/><br/>
+> You should handle all documented and unknown errors how you see fit and validate if a payment is payable before making requests to avoid a bad user experience.
+
+```mermaid
+sequenceDiagram
+  autonumber
+
+	actor user
+	participant t as Terminal/POS
+	participant a as App
+	participant b as Integrator's Backend
+	participant c as Centrapay
```

Comment:
@IsaacMurtagh These aliases are an anti-pattern that we've tried to avoid in the past - suggest using the full name to make this easier to maintain